src/OpenLoco/src/Ui/WindowManager.cpp

```diff
@@ -2007,6 +2007,12 @@ namespace OpenLoco::Ui::WindowManager
                     flagsChanged = true;
                 }
 
+                if (viewport->hasFlags(ViewportFlags::seeThroughBridges))
+                {
+                    viewport->flags &= ~(ViewportFlags::seeThroughBridges);
+                    flagsChanged = true;
+                }
+
```

Comment:
we really should add a helper method to reduce this boilerplate code, but that's for another time

Reply:
Noted